rules/windows/collection_posh_audio_capture.toml

```diff
@@ -17,31 +17,42 @@ note = """## Triage and analysis.
 
 ### Investigating PowerShell Suspicious Script with Audio Capture Capabilities
 
-PowerShell is one of the main tools used by system administrators for automation, report routines, and other tasks.
+PowerShell is one of the main tools system administrators use for automation, report routines, and other tasks, making
+it available for use in various environments, creating an attractive way for attackers to execute code.
 
-Attackers can use PowerShell to interact with the Windows API and capture audio from input devices connected to the
-computer.
+Attackers can use PowerShell to interact with the Windows API with the intent of capturing audio from input devices
+connected to the victim's computer.
 
-#### Possible investigation steps:
+#### Possible investigation steps
 
 - Examine script content that triggered the detection. 
-- Investigate script execution chain (parent process tree).
+- Investigate the script execution chain (parent process tree).
 - Inspect any file or network events from the suspicious PowerShell host process instance.
-- If the action is suspicious for the user, check for any other activities done by the user in the last 48 hours.
+- Investigate other alerts related to the user/host in the last 48 hours.
+- Check whether it makes sense for the user to use PowerShell to complete its tasks.
```

Comment:
```suggestion
- Consider whether the user needs PowerShell to complete its tasks.
```

Reply:
Just leaving this once so feel free to do a search and replace if you decide to incorporate this edit :)

Reply:
https://github.com/elastic/detection-rules/pull/1805/commits/375605f2b330c2fa679f2c5fe0b5c336ac2c726e